ui/media/main.js

```diff
@@ -277,16 +299,40 @@ function playSound() {
 
 async function healthCheck() {
     try {
-        let res = await fetch('/ping')
-        res = await res.json()
-
-        if (res[0] == 'OK') {
-            setStatus('server', 'online', 'success')
+        let res = undefined
+        if (sessionId) {
```

Comment:
Will this ever be false? Do we need this `if` since we generate the `sessionId` at the top?